bundles/asdi/asdi-projection-change/plugin/ProjectionChangerPlugin.js

```diff
@@ -0,0 +1,101 @@
+Oskari.clazz.define( 'Oskari.projection.change.ProjectionChangerPlugin',
+  function ( config, instance ) {
+    this._instance = instance;
+    this._config = config || {};
+    this._clazz = 'Oskari.projection.change.ProjectionChangerPlugin';
+    this._defaultLocation = 'top right';
+    this._index = 55;
+    this._visible = false;
+    this._templates = {
+      projectionchanger: jQuery('<div class="mapplugin oskari-projection-changer"></div>')
+    };
+    this._loc = instance.getLocalization();
+    this._mobileDefs = {
+      buttons:  {
+          'mobile-projectionchanger': {
+              iconCls: 'mobile-projectionchange',
+              tooltip: '',
+              show: true,
+              callback: function () {
+              },
+              sticky: true,
+              toggleChangeIcon: false
+          }
+      },
+      buttonGroup: 'mobile-toolbar'
+    };
+    this._flyout = this._instance.getFlyout();
+    this._flyout.hide();
+    this._log = Oskari.log('Oskari.projection.change.ProjectionChangerPlugin');
+  }, {
+
+    _createControlElement: function () {
+        var launcher = this._templates.projectionchanger.clone();
+
+        launcher.attr('title', this._loc.tooltip.tool);
+
+        if ( this._config.noUI ) {
+            return null;
+        }
+      return launcher;
+    },
+    createUi: function() {
+      this._element = this._createControlElement();
+      this.handleEvents();
+      this.addToPluginContainer(this._element);
+    },
+    handleEvents: function () {
+        var me = this;
+        this._flyout.move(1300, 120, true);
+        this.getElement().on( "click", function() {
+            if ( !me._visible ) {
```

Comment:
You can use flyout.isVisible() instead or even add a toggle() function for ExtraFlyout.